.github/workflows/release.yaml

```diff
@@ -55,10 +55,10 @@ jobs:
           commit_username: validator-labs-bot
           commit_email: bot@noreply.validator-labs.io
       - name: Trigger docs.crd.dev index
-        if: ${{ needs.release-please.outputs.tag_name }}
+        if: needs.release-please.outputs.releases_created == 'true'
         run: |
-          curl -v https://doc.crds.dev/github.com/${{ github.event.repository.owner.login }}/${{ github.event.repository.name }}@${{ github.event.inputs.tag }}
-          echo "Triggered docs.crd.dev to index ${{ github.event.repository.name }}@${{ github.event.inputs.tag }}"
+          curl -v https://doc.crds.dev/github.com/${{ github.event.repository.owner.login }}/${{ github.event.repository.name }}@${{ github.event.outputs.tag_name }}
+          echo "Triggered docs.crd.dev to index ${{ github.event.repository.name }}@${{ github.event.outputs.tag_name }}"
```

Comment:
Please justify why you are using `${{ github.event.outputs.tag_name }}` instead of `${{ needs.release-please.outputs.tag_name }}` when this workflow uses the latter everywhere else and we know it's correct.

Have you tested this?

Reply:
Sorry, thanks for catching that too.

I manually tested the concepts used here in a personal repo, and then copied things over here when I was satisfied that things worked properly. But it was copied over incorrectly. It should be good now.

I can test this PR too if I can find a way to.